app/presenters/search_presenters.rb

```diff
@@ -124,10 +124,12 @@ def item
     end
 
     def options
-      super.merge(:classes => [Topic, CMS::Page])
+      super.merge(:classes => [CMS::Page])
     end
 
     def search
+      return @search if @search
```

Comment:
Hmm... I don't understand this change... Would the rest of the method ever execute? :thinking:  (i.e. in which cases `@search` would not be present?)

Reply:
just normal memoization, before method is called for the first time, the variable would be `nil`